pkgs/servers/mail/postfix/pflogsumm.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, buildPerlPackage, perlPackages
+}:
+
+buildPerlPackage rec {
+  name = "pflogsumm-${version}";
+  version = "1.1.3";
+
+  src = fetchurl {
+    url = "http://jimsun.linxnet.com/downloads/${name}.tar.gz";
+    sha256 = "0hkim9s5f1yg5sfs5048jydhy3sbxafls496wcjk0cggxb113py4";
+  };
+
+  outputs = [ "out" ];
+  propagatedBuildOutputs = [ ];
+
+  buildInputs = [ perlPackages.DateCalc ];
+
+  preConfigure = ''
+    touch Makefile.PL
+    patchShebangs .
+  '';
+  doCheck = false;
```

Comment:
Hm, it is the default for `stdenv.mkDerivation` but evidently, `buildPerlPackage` deviates.